<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportPoints</class>
 <widget class="QDialog" name="ImportPoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Point File(s)</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="SelectFileL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected File(s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BrowseLE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="PointNameL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Name =</string>
   </property>
  </widget>
  <widget class="QLabel" name="XL">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X =</string>
   </property>
  </widget>
  <widget class="QLabel" name="YL">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y =</string>
   </property>
  </widget>
  <widget class="QLabel" name="ZL">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z =</string>
   </property>
  </widget>
  <widget class="QLabel" name="ExplanationL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Specify Point File Format</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PointNameLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="XLE">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="YLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZLE">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ExplanationL2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Point Group</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CreateGroupChB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Group</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SubGroupListCB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
